lua/aerial/highlight.lua

```diff
@@ -29,6 +29,7 @@ M.create_highlight_groups = function()
   -- The name of the symbol
   for _, symbol_kind in ipairs(M.identifiers) do
     link(string.format("Aerial%s", symbol_kind), "NONE")
+    vim.cmd(string.format("hi Aerial%sPrivate cterm=italic gui=italic", symbol_kind))
```

Comment:
Would prefer for this to be the native lua function, and also to set the "default" highlight so it doesn't override settings that the user may already have in place
```lua
vim.api.nvim_set_hl(0, 'AerialPrivate', { default = true, italic = true})
```